src/nats.h

```diff
@@ -1045,6 +1045,20 @@ natsOptions_SetPingInterval(natsOptions *opts, int64_t interval);
 NATS_EXTERN natsStatus
 natsOptions_SetMaxPingsOut(natsOptions *opts, int maxPingsOut);
 
+/** \brief Sets the size of the internal read buffer.
+ *
+ * Sets the size, in bytes, of the internal read buffer used for read data from
```

Comment:
Maybe `used for reading data from` or `used to read data from`?

Reply:
Correct, but I get the feeling that it would be better to describe write things.
How about this?

```
/** \brief Sets the size of the internal read/write buffers.
 *
 * Sets the size, in bytes, of the internal read/write buffers used for reading/writing data from
 * a socket.
 * If not specified, or the value is 0, the library will use a default value,
 * currently set to 32KB.
 *
 * @param opts the pointer to the #natsOptions object.
 * @param ioBufSize the size, in bytes, of the internal buffer for read/write
 * operations.
 */
```

Reply:
See 30984fd.